spotiplex/modules/spotiplex/main.py

```diff
@@ -84,6 +85,18 @@ def process_playlist(self: "Spotiplex", playlist: str) -> None:
             playlist_name: str | None = self.spotify_service.get_playlist_name(
                 playlist_id,
             )
+            if (self.include_playlist_author):
+                playlist_author: str | None = self.spotify_service.get_playlist_author(
+                    playlist_id,
+                )
+
+                parts = playlist_author.split()
+                first_name = parts[0]
+                last_name = parts[-1]
+                last_initial = last_name[0]  # Get the first letter of the last name
+                short_author = f"{first_name} {last_initial}."
+
+                playlist_name = f"{playlist_name} [{short_author}]"
```

Comment:
```suggestion
            if (self.include_playlist_author):
                playlist_author: str = self.spotify_service.get_playlist_author(
                    playlist_id,
                )
                if playlist_author is "Unknown":
                    playlist_name = f"{playlist_name} [{playlist_author}]"
                else:
                    parts = playlist_author.split()
                    first_name = parts[0]
                    last_name = parts[-1]
                    last_initial = last_name[0]  # Get the first letter of the last name
                    short_author = f"{first_name} {last_initial}."

                    playlist_name = f"{playlist_name} [{short_author}]"
```

This should make it more adaptable and not cause a failure if we can't pull the name for some reason.

Reply:
Very good point! I'd personally keep `get_playlist_author` returning None and change the following as shown below:

This way if we can get the playlist author it shows `My Playlist Name [Author]`, or just `My Playlist Name` if we can't  
 
```suggestion
            if (self.include_playlist_author):
                playlist_author: str | None = self.spotify_service.get_playlist_author(
                    playlist_id,
                )

                if playlist_author is not None:
                    parts = playlist_author.split()
                    first_name = parts[0]
                    last_name = parts[-1]
                    last_initial = last_name[0]  # Get the first letter of the last name
                    short_author = f"{first_name} {last_initial}."

                    playlist_name = f"{playlist_name} [{short_author}]"
```

Let me know what you think

Reply:
This is better, but users will expect an author if they have the setting enabled, so add a logger.error if the author is none with the playlist name and that should be good I think 🙂

Something like `logger.error(f"Unable to get author for {playlist_name})`?

Reply:
that makes sense to me. I'll fix that up